gui/src/App.tsx

```diff
@@ -235,8 +238,8 @@ export default function App() {
       }
     }
 
-    document.addEventListener('keypress', onKeyboard);
-    return () => document.removeEventListener('keypress', onKeyboard);
+    document.addEventListener('keyup', onKeyboard);
+    return () => document.removeEventListener('keyup', onKeyboard);
```

Comment:
Please just move this into a different PR...

Reply:
*makes 4 branches*